### DISCUSSION TITLE
Optimization / Make the process faster

### DISCUSSION THREAD

**Discussion by ubanning** [PaddlePaddle/PaddleOCR#10423] [Q&A]
Hi everyone, I'm a beginner and I need help. I need to analyze several images using PaddleOCR and I want to make it as fast as possible. I have a GPU with several GB of vRAM, but it uses a maximum of 1.5GB, is there any way I can use more of my GPU to process images faster? Parallel computing? I've tried changing parameters like max_batch_size, rec_batch_num and I didn't get any difference! Can anybody help me? How can I increase the speed? Or the only way is to buy more GPUS and using only 15% of them :(
Thanks.

**Comment by wesleyroseno** [comment]
@ubanning Did you manage to make the process faster?

**Comment by HawkingRadiation42** [comment]
were you able to do this?

**Comment by zovelsanj** [comment]
@ubanning did you solve it? Did you use `paddlepaddle-gpu`?
